references/cli/dryrun.go

```diff
@@ -88,9 +96,10 @@ You can also specify a remote url for app:
 		},
 	}
 
-	cmd.Flags().StringVarP(&o.ApplicationFile, "file", "f", "./app.yaml", "application file name")
+	cmd.Flags().StringSliceVarP(&o.ApplicationFiles, "files", "f", []string{"app.yaml"}, "application related file names")
```

Comment:
```suggestion
	cmd.Flags().StringSliceVarP(&o.ApplicationFiles, "file", "f", []string{"app.yaml"}, "application related file names")
```

We should keep the compatiblity, and naming it `file` with `--file filea --file fileb` is also reasonable